frontend/components/transaction/BidTransaction.stories.js

```diff
@@ -93,41 +73,27 @@ const common = {
                 this.isExecuting = false;
             }, 1000);
         },
-        setTransactionAmountDAI(amount) {
-            this.transactionAmountDai = amount;
-        },
-    },
-    watch: {
-        transactionAmountDai(newAmount) {
-            if (newAmount === undefined) {
-                this.transactionAmountDai = this.auctionTransaction.totalPrice;
-            }
+        deposit() {
+            this.walletVatDai = this.transactionAmountDai;
         },
     },
     template: `
     <BidTransaction
         :auctionTransaction="auctionTransaction"
         :isConnecting="isConnecting"
-        :isGrantingAccess="isGrantingAccess"
-        :isDepositing="isDepositing"
```

Comment:
We still may want to provide `isDepositing` which will turn `Manage DAI in VAT` button into the loading state in case user closed the wallet modal before the transaction finished.